lms/djangoapps/course_api/views.py

```diff
@@ -0,0 +1,53 @@
+"""
+Course API Views
+"""
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.lib.api.view_utils import view_auth_classes
+from xmodule.modulestore.django import modulestore
+
+
+@view_auth_classes()
+class CourseView(APIView):
+    """
+    Course API view
+    """
+
+    def get(self, request, course_key_string):
+        """
+        Request information on a course specified by `course_key_string`.
+            Body consists of a `blocks_url` that can be used to fetch the
+            blocks for the requested course.
+
+        Arguments:
+            request (HttpRequest)
+            course_key_string
+
+        Returns:
+            HttpResponse: 200 on success
```

Comment:
This needs to be something more descriptive about the return item otherwise for a view, returning a 200 would be assumed.